napari/_qt/_tests/test_qt_viewer.py

```diff
@@ -336,28 +336,11 @@ def test_qt_viewer_clipboard_with_flash(make_napari_viewer, qtbot):
     clipboard_image = QGuiApplication.clipboard().image()
     assert clipboard_image.isNull()
 
-    # capture screenshot
```

Comment:
Why are we calling `clipboard` twice? Can't we just remove the deprecated version?